src/widget/wsearchlineedit.cpp

```diff
@@ -798,3 +807,11 @@ void WSearchLineEdit::slotSetFont(const QFont& font) {
         updateClearAndDropdownButton(getSearchText());
     }
 }
+
+bool WSearchLineEdit::hasSelectedText() const {
+    const QLineEdit* pEdit = lineEdit();
+    VERIFY_OR_DEBUG_ASSERT(pEdit) {
+        return false;
+    }
+    return pEdit->hasSelectedText();
+}
```

Comment:
As I already pointed out earlier, this is obsolete:
There is always a QLineEdit because we set the combobox editable in the constructor, and we rely on that in many places here. Please see
https://github.com/qt/qtbase/blob/4903e45231b00fb8a06f0b3c60af764f13e831bd/src/widgets/widgets/qcombobox.cpp#L1787-L1802

Just use lineEdit() directly.

Also applies here https://github.com/mixxxdj/mixxx/pull/11289/commits/7165f0b51097c45bc9f99089a646c22533d54e95#diff-85c225cd63139457cde84ddb0d2f69c854c16f8254672cd19587fd25dd755c62R316

Reply:
The other changes LGTM, thanks!